### PR TITLE
Homework2: Evgeni Kuleuski

### PR DESCRIPTION
**Pull request by evgenii-kylevski**

### CODE REVIEW

File: Homework2.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+# Задание 1: программа считает сумму заказа
+
+m = int(input("Введите колличество рублей: "))
+n = int(input('введите колличество копеек: '))
+k = int(input('Количество покупок '))
+n = n / 100
+l = m + n
+all = l * k
```
Comment (belovmd) [inline Homework2.py]: all is built-in function and it seems to be bad name for variable

File: Homework2.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+all_c = round(all // 1)
+all_p = round((all % 1) * 100)
+print('Спасибо, что сделали покупку у нас.')
+print('Общая цена ', all_c, ' рублей ', all_p, ' копеек.')
+
+# Задача 1
+# Выведите все элементы, которые меньше 5
+a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+for element in a:
+    if 5 < element < 25:
```
Comment (belovmd) [inline Homework2.py]: this code contradicts with task description: if we should print numbers less than 5 this line isn't correct. If this line is correct we should fix description

File: Homework2.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+# общих для этих двух списков.
+
+a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+b = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+
+stack = []
+
+for element in a:
+    if element in b:
+        stack.append(element)
```
Comment (belovmd) [inline Homework2.py]:
```suggestion
[element for element in a if element in b]
```

File: Homework2.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+# дополнительное задание
+# Определите, является ли число палиндромом (читается слева
+# направо и справа налево одинаково).
+# Число положительное целое, произвольной длинны
+
+a_int = input('Введите любое положительное число: ')
+
+a_int = a_int.replace(' ', '')
+
+if a_int == a_int[::-1]:
```
Comment (belovmd) [inline Homework2.py]: Please don't use strings in this task

**Review by belovmd — CHANGES_REQUESTED** [review]
Thank you for your work. 
Please address these issues

**Comment by evgenii-kylevski** [comment]
all notes taken into account

**Review by belovmd — COMMENTED** [review]
8

**Pull request closed without merging**